test/api/routes/movies.spec.js

```diff
@@ -1,90 +1,87 @@
-const supertest = require("supertest")
-const app = require("../../../src/server.js")
-const { createMovie } = require("../../helpers/createMovie.js")
-const { createScreen } = require("../../helpers/createScreen.js")
+const supertest = require('supertest')
```

Comment:
having a different linter set up for the spacing in the code makes it a little difficult to see what exactly you have modfied

Reply:
I actually didn't touch this part, not sure why we have this here. 🤔

Reply:
I think its just because your local prettier or equivalent has different settings to the linter in the file, and when you've formatted it's added in a bunch of extra tabs 🙃

Reply:
Oh yeah, right🤦🏻‍♀️ Will look into that